### ISSUE TITLE
Train model with equivalent performance from scratch

### ISSUE THREAD

**Issue by wingman-jr-addon** [wingman-jr-addon/wingman_jr#24]
Goal: try training a 300x300 network and see if equivalent performance can be met. This will take a while with current training rig.

**Comment by wingman-jr-addon** [comment]
I have had some thermal-off difficulties with my TX1 and have had to crank things back a bit. Seems to be rolling now after several false starts.

**Comment by wingman-jr-addon** [comment]
Had further difficulties with that first batch of training due to crashes. Stabilized training process further. In progress, but may need some experimentation with hyperparameters before I know if we can reach ImageNet-level accuracy.

**Comment by wingman-jr-addon** [comment]
Still training...

**Comment by wingman-jr-addon** [comment]
This peaked out 67.5% or so. Not fantastic, but within 10% of finetuning.

**Comment by wingman-jr-addon** [comment]
I'd like to try this again, but it will take quite some time. One thing I'm checking out also is how the performance of the ROC curve (e.g. as in #29 ) fairs with a lighter model, perhaps say MobileNetV2 224 0.35. Need to mull this over a bit.

**Comment by wingman-jr-addon** [comment]
One of the key blockers for this was obtaining good enough hardware. I have now gotten my GPU server up and running ([see the writeup on my blog](https://wingman-jr.blogspot.com/2020/03/the-quest-for-new-hardware-pt-1.html) ). I just need to setup my "rack" and move it, so this is currently waiting on that.

**Comment by wingman-jr-addon** [comment]
Ok, GPU server is in place. I've started training MobileNetV2 from scratch. Early attempts see near-parity for performance, with raw accuracy tapping out at 65%. Working on changing tuning parameters to better account for "from scratch" and with much larger batch sizes.

**Comment by wingman-jr-addon** [comment]
I've tuned the training significantly and am seeing MobileNetV2 train from scratch to around 72% so I am calling this successful.
However, I'm not seeing a boost by having fully trained the network from scratch without having the clash between e.g. the ImageNet scratch-training domain and the NSFW-data- finetuning-training domain.
In some sense that means the overall intent was a failure in that we won't see an improvement this way, but that just leads to #64 .
